src/components/FilterTags/_filter-tags.scss

```diff
@@ -5,7 +5,9 @@
   padding: 0.5rem;
   position: relative;
 
-  &__input {
+  // Increased specificity needed here due to:
+  // https://github.com/canonical-web-and-design/jaas-dashboard/issues/254
+  .p-filter-tags__input {
```

Comment:
Could this be `& &__input {` ?

Reply:
Yes but I prefer this approach as it's more verbose and easier to read.